hazelcast/src/main/java/com/hazelcast/client/impl/clientside/ClientSchemaService.java

```diff
@@ -78,18 +79,18 @@ public void putLocal(Schema schema) {
         putIfAbsent(schema);
     }
 
-    private boolean putIfAbsent(Schema schema) {
+    private void putIfAbsent(Schema schema) {
         long schemaId = schema.getSchemaId();
         Schema existingSchema = schemas.putIfAbsent(schemaId, schema);
```

Comment:
is it a good idea to have eviction on this `schemas` map? No limit for its grow currently.

Reply:
Yes, we don't have the concept of deleting a schema, and we should be aware of all the schemas we use in the client side and all the schemas in the cluster in the member side.

We assume that the number of schemas will be low